pyproject.toml

```diff
@@ -63,8 +68,8 @@ requires = [
   # version.  See:
   # https://numpy.org/doc/stable/release/1.25.0-notes.html#compiling-against-the-numpy-c-api-is-now-backwards-compatible-by-default
   "numpy>=1.25; python_version > '3.8'",
-  # NEP29-minimum as of Sep 21, 2023
-  "numpy==1.22; python_version >= '3.7' and python_version < '3.9'",
+  # SPEC0-minimum as of Dec 23, 2023
+  "numpy==1.22; python_version <= '3.8'",
```

Comment:
Shouldn't this be `python_version >= '3.8'"`?

Reply:
No. Python 3.9 is supported by numpy 1.25, so it's covered by the previous line. I could set this `== '3.8'`, but I'm not 100% sure what the result would be if someone tried to build with Python 3.7, as it would think it could build without numpy at all. OTOH, it's possible that the `python-requires` would prevent from getting that far.

Reply:
Ok - yes - sorry - I remember.